assets/sass/components/global/_googlesitekit-badge.scss

```diff
@@ -23,6 +23,7 @@
 	display: inline-block;
 	font-family: $f-primary;
 	font-size: 10px;
+	font-weight: 500;
```

Comment:
This was added to avoid variation when left undefined where it would be (and was) inherited. In titles this usually left it with a heavier weight and other places it had a normal weight. Looking at similar components in the wild (in GitHub for example), a heavier weight is used, likely for better contrast/readability, especially with darker backgrounds so this makes sense to have on the base class.